src/wp-includes/blocks/comment-template.php

```diff
@@ -58,11 +58,10 @@ function block_core_comment_template_render_comments( $comments, $block ) {
 				$block_content .= sprintf( '<ol>%1$s</ol>', $inner_content );
 				--$comment_depth;
 			} else {
-				$inner_content  = block_core_comment_template_render_comments(
+				$block_content .= block_core_comment_template_render_comments(
```

Comment:
When the inner content contains the placeholder symbols (e.g. `%s`, `%1$s`, etc.), the following happens with the original code:

* PHP 8+ `Fatal error: Uncaught ArgumentCountError:`
* < PHP 8 `Warning: sprintf(): Too few arguments`

https://3v4l.org/KabQ1

See https://www.php.net/manual/en/function.sprintf.php